### ISSUE TITLE
nan problem

### ISSUE THREAD

**Issue by pzhren** [noureldien/timeception#12]
How to solve the nan problem in the test

**Comment by noureldien** [comment]
Hi, thanks for asking. I didn't experience any nan problem. Maybe you have a high learning rate.

**Comment by noureldien** [comment]
The metric function I got it from here
https://github.com/gsig/charades-algorithms/blob/master/pytorch/utils/map.py

**Comment by noureldien** [comment]
Hi,
Please note that for testing Charades, you only evaluate the entire test set, at the end of each training epoch. So, you can modify the code and define a new callback (for testing at the end of each epoch) and pass it to model.fit_generator.


Nour
